src/test/runner.ts

```diff
@@ -40,7 +40,7 @@ const removeFolderAsync = promisify(rimraf);
 const readDirAsync = promisify(fs.readdir);
 const readFileAsync = promisify(fs.readFile);
 
-type RunResult = 'passed' | 'failed' | 'sigint' | 'forbid-only' | 'no-tests' | 'timedout';
+type RunResult = 'passed' | 'failed' | 'sigint' | 'forbid-only' | 'no-unique-spec-titles' | 'no-tests' | 'timedout';
```

Comment:
`clashing-spec-titles`?

Reply:
I was looking for a word != unique. Clashing sounds better!